package.json

```diff
@@ -19,7 +19,7 @@
     "node": ">= 6"
   },
   "scripts": {
-    "install": "lerna bootstrap",
+    "install": "lerna bootstrap --hoist --no-ci",
```

Comment:
Should not these options be in the package.json? (https://github.com/lerna/lerna#lernajson)